src/lib/types.ts

```diff
@@ -1,16 +1,19 @@
 export type Provider =
```

Comment:
we also have 2 new sms providers (vonage & textlocal) and another oauth provider (linkedin), can we add those in as well?